disnake/enums.py

```diff
@@ -812,6 +813,12 @@ class ThreadSortOrder(Enum):
     creation_date = 1
 
 
+class ThreadLayout(Enum):
+    not_set = 0
+    list_view = 1
+    gallery_view = 2
```

Comment:
It would be nice to have `list` and `gallery` aliases, in my opinion

Reply:
Aliases seem a bit much for such a fairly insignificant field; I've updated the enum in 6b64137322a593892a602ece1b5523629d6801fe to use those names instead, but haven't completely decided yet.
Enum members are usually named according to the api docs, so we may want to consider keeping the original ones. I'll keep this discussion open for now.

Reply:
I'm fine with using the API names, I don't think people will complain about it🤷 

You can resolve this :)